src/cmakeFindModules/FindOpenImageIO.cmake

```diff
@@ -0,0 +1,18 @@
+#find headers
```

Comment:
this one seems to have EXIF R/W too
http://freeimage.sourceforge.net/index.html